src/medCore/gui/medSelectorWorkspace.h

```diff
@@ -0,0 +1,43 @@
+/*=========================================================================
+
+ medInria
+
+ Copyright (c) INRIA 2013 - 2014. All rights reserved.
+ See LICENSE.txt for details.
+
+  This software is distributed WITHOUT ANY WARRANTY; without even
+  the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+  PURPOSE.
+
+=========================================================================*/
+
+#pragma once
+
+#include <medAbstractWorkspace.h>
+
+class medSelectorWorkspacePrivate;
+class medSelectorToolBox;
+
+/**
+ * class medSelectorWorkspace
+ * Defines the Selector workspace.
+ */
+class medSelectorWorkspace : public medAbstractWorkspace
+{
+    Q_OBJECT
+
+public:
+    medSelectorWorkspace(QWidget * parent, QString name);
+
+    virtual ~medSelectorWorkspace();
+
+    QPointer<medSelectorToolBox> selectorToolBox();
+
+protected slots:
+    virtual void onProcessSuccess(){}
```

Comment:
I'm still bothered by the choice of name here. (BTW, making a change in response to a comment and then reverting it through another commit isn't a good idea)
Slots are like regular functions, which should be named according to what they do, not why they are called. Imagine a function that displays an image you have just loaded. You would call that function ```displayImage```, not ``onLoadedImage```.
By naming QT signals after the event that triggered them, and the slots by the action they do, it becomes much easier to understand the code.
Compare:
```
connect(process, SIGNAL(onSuccess), this, SLOT(onProcessSuccess))
```

with
```
connect(process, SIGNAL(onSuccess), this, SLOT(importOutput))
```